spec/mongoid/scroll_cursor_spec.rb

```diff
@@ -8,6 +8,16 @@
     its(:tiebreak_id) { should be_nil }
     its(:value) { should be_nil }
     its(:criteria) { should eq({}) }
+    describe 'base64' do
```

Comment:
I'm not sure it follows the spec style of this gem. Feel free to correct me, and I'll add other specs after your feedback.